databox/api/src/Attribute/AttributeManager.php

```diff
@@ -21,4 +21,11 @@ public function getAttributeDefinitionBySlug(string $workspaceId, string $slug):
             'workspace' => $workspaceId,
         ]);
     }
+
+    public function getAttributeDefinitions(string $workspaceId): iterable
```

Comment:
Can you please move all the methods of this class to `AttributeDefinitionRepository`?